slc7-builder/Dockerfile.arm64

```diff
@@ -25,7 +25,7 @@ RUN yum install -y glibc coreutils bash tcsh zsh perl tcl tk readline \
                    readline-devel perl-Switch perl-Storable perl-Env packages \
                    perl-Thread-Queue  tk-devel tcl-devel which python2-pip \
                    file git subversion bc finger zip unzip nano hostname \
-                   vim wget screen time && yum clean all
+                   vim wget screen time voms-clients-cpp && yum clean all
```

Comment:
Ignore my previous comment, you can bind-mount within the container the certificates and vomses from here: `/cvmfs/grid.cern.ch/etc/grid-security`